eadf/__init__.py

```diff
@@ -160,7 +160,10 @@
 
 from .importers import fromAngleListData, fromFourierData, fromArraydefData
 
-from .importers import fromIrregularAngDataMatFile
+from .importers import (
+    fromIrregularAngDataNBandMatFile,
```

Comment:
What is so irregular about the angular data anyway? I don't understand the function name, but I also cannot offer anything better.

Reply:
By irregular, it meant not in the regular format expected for the EADF.

Reply:
Ah, I see. In that case this is not necessary, bc if it were regular in that sense we would not write an importer anyway.

Reply:
Updated to fromWidebandAngDatMatFile and fromNarrowbandAngDatMatFile.